smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiConfig.java

```diff
@@ -218,6 +219,14 @@ public void setForbidGreedyLabels(boolean forbidGreedyLabels) {
         this.forbidGreedyLabels = forbidGreedyLabels;
     }
 
+    public boolean getRemoveGreedyParameterSuffix() {
```

Comment:
This should have a guide update, since it's a new option.

Reply:
Addressed in ee69c1c.